assets/scss/components/_wishlist-list.scss

```diff
@@ -31,3 +31,12 @@
     gap: 12px;
     align-items: center;
 }
+
+.c-wishlist-list__icon {
+    width: rem-calc(24px);
+    height: rem-calc(24px);
+
+    use {
+        fill: transparent;
```

Comment:
You can set `fill: transparent` directly in `c-wishlist-list__icon`, no need to nest `use` here